Obsidian/Commands/MainCommandModule.cs

```diff
@@ -49,29 +50,36 @@ public async Task PluginsAsync()
             };
 
             var messages = new List<ChatMessage>();
-
+            int plugins_i = 0;
             foreach (var pluginContainer in Context.Server.PluginManager.Plugins)
+            {
```

Comment:
could be changed to a simple for instead of a foreach

Reply:
```suggestion
            for (int i=0;i<Context.Server.PluginManager.Plugins.Count;i++)
            {
                   var pluginContainer = Context.Server.PluginManager.Plugins[i];
```
like this?

Reply:
Yep

Reply:
Ok, I'll change it.